docs/local-dev/cli-build-system.md

```diff
@@ -592,19 +592,26 @@ A complete launch configuration for VS Code debugging may look like this:
 ```json
 {
   "type": "node",
-  "name": "vscode-jest-tests",
+  "name": "vscode-jest-tests.v2",
   "request": "launch",
+  "env": {
+    // Needed to disable watch by default mode.
+    "ENV": "CI"
```

Comment:
This should already be disabled through the `--watchAll=false` option, or does that not work?

Reply:
Ah yup, will remove.